SConstruct

```diff
@@ -222,6 +238,20 @@ def abspath(x):
     # rpath works elsewhere
     return x[0].path.rsplit("/", 1)[1][:-3]
 
+#Cython build enviroment
+envB = env.Clone()
+
+added_libs = []
+if arch == "Darwin":
+  envB["LINKFLAGS"]=["-bundle", "-undefined", "dynamic_lookup"]
+elif arch == "aarch64":
+  envB["LINKFLAGS"]=["-shared"]
+
+  added_libs.append(python_path.split("/")[-1])
+else:
+  envB["LINKFLAGS"]=["-pthread", "-shared"]
+
+Export('envB', 'added_libs')
```

Comment:
Is it possible to do it wihout envB? If you use `SharedLibrary` instead of `Program` you probably don't need those flags.

If we need to can you give `envB` and `added_libs` are clearer name?